README.md

```diff
@@ -360,6 +361,62 @@ public void Configure(IApplicationBuilder app)
 }
 ```
 
+### 2.8. Typed Handlers
```

Comment:
@ben-foster-cko Have a read through these docs and let me know what you think. Cheers.

Reply:
Really nice!